Resources/Private/FormFramework/Standard/Frontend/Partials/Checkboxlink.html

```diff
@@ -0,0 +1,28 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" xmlns:formvh="http://typo3.org/ns/TYPO3/CMS/Form/ViewHelpers" data-namespace-typo3-fluid="true">
```

Comment:
Very long line. Please add some returns after each attribute like you have done with f:form.checkbox in line 6

Reply:
Done